src/Plugin/Block/ContactSummaryTab.php

```diff
@@ -105,29 +105,6 @@ public function build() {
       $build['#content']['left']['content']['#title'] = $block->label();
     }
 
```

Comment:
As per comment on the tab config entity, I think this should be gone entirely now...